libraries/importers/component-importer/src/main/java/org/eclipse/sw360/importer/ComponentCSVRecord.java

```diff
@@ -288,6 +286,34 @@ private boolean isSetClearingInformation() {
         return false;
     }
 
+    private boolean isSetEccInformation() {
```

Comment:
**I am OK with that,** but the code is still very ugly and one should refactor it. Nearly every line is duplicated code here, and it is hard to read due to the negations ("if not `eccStatus` is empty return true else if not `eccAl` is empty return true else [....] else return false" is harder to understand than "return ( not all are empty )").